setup.py

```diff
@@ -24,6 +24,7 @@ def test_suite():
         'ply',
         'rdflib',
         'six',
+        'ruamel.yaml',
```

Comment:
Please do not use that YAML parser: it requires a native and a C compiler. Use PyYaml instead